modules/chemical_reactions/test/tests/jacobian/tests

```diff
@@ -51,12 +51,12 @@
     type = PetscJacobianTester
     input = 2species_equilibrium.i
     ratio_tol = 1e-7
-    difference_tol = 5e-7
+    difference_tol = 1e-6
```

Comment:
I guess Petsc changed the way they do their finite differencing? I'm surprised this doesn't show up in any `AnalyzeJacobian` tests

Reply:
This test was only added very recently, so the tolerances were tightly tuned to some recent version of PETSc. IIRC it was also very close to passing with PETSc 3.4.5 as it was so I don't think anything too major changed.